lib/config/fileconf.go

```diff
@@ -1090,6 +1092,12 @@ type DatabaseTLS struct {
 	CACertFile string `yaml:"ca_cert_file,omitempty"`
 }
 
+// DatabaseOptions are an additional database options.
+type DatabaseOptions struct {
+	// MySQLServerVersion is the MySQL version reported by DB proxy instead of default Teleport string.
+	MySQLServerVersion string `yaml:"mysql_server_version"`
```

Comment:
Should we group (adding an inner struct) options per database?
Or generalize the naming here to `ServerVersion` `ReportedServerVersion`

Reply:
>Should we group (adding an inner struct) options per database?
This configuration is already added per database. https://github.com/gravitational/teleport/pull/12196/files#diff-7e45ea46c6e1e877dc254cc472ea0f99adb1853b8d2a675bde739c5c08289310R115-R117

> Or generalize the naming here to ServerVersion ReportedServerVersion
The version reported by the Database is not a part of the Teleport configuration. Look here: https://github.com/gravitational/teleport/blob/0e992e138fb232d9b12b4d78c2f221f7d13cc907/api/types/types.proto#L234-L237
Database resource has two fields, `Spec` and `Status`. The version set in database configuration ends up in the `Spec` struct. The version reported by the DB end up in `Status` struct which is dynamic, so there is already a distinction between those two versions.